tests_e2e/scenario_utils/logging_utils.py

```diff
@@ -0,0 +1,33 @@
+# Create a base class
```

Comment:
When I fixed the dependencies, I realized I missed this file in my previous PR. This file comes from the current Azure Pipeline and needs to be rewritten